src/themes/five/src/components/Checkout/index.js

```diff
@@ -97,6 +97,15 @@ const CheckoutUI = (props) => {
 
   const isWalletEnabled = configs?.wallet_enabled?.value === '1' && (configs?.wallet_cash_enabled?.value === '1' || configs?.wallet_credit_point_enabled?.value === '1')
 
+  const isDisablePlaceOrderButton = !cart?.valid ||
+    (!paymethodSelected && cart?.balance > 0) ||
+    placing ||
+    errorCash ||
+    loading ||
+    !cart?.valid_maximum ||
+    (!cart?.valid_minimum && !(cart?.discount_type === 1 && cart?.discount_rate === 100)) ||
+    ([3, 4].includes(options?.type) && !cart?.place)
```

Comment:
please don't use this validation ([3, 4]) because is hardcoded, create a constant for this